mlflow/utils/search_logged_model_utils.py

```diff
@@ -92,7 +93,14 @@ def parse_filter_string(filter_string: Optional[str]) -> list[Comparison]:
         )
 
     comparisons: list[sqlalchemy.BinaryExpression] = []
-    for stmt in parsed[0]:
+    statements = [s for s in parsed[0] if not s.is_whitespace]
```

Comment:
Can we reuse some logic in `_join_in_comparison_tokens`? This implementation looks good though :)

Reply:
it works!